core/src/actors/session/handlers.rs

```diff
@@ -208,7 +208,20 @@ impl Handler<SendBlock> for Session {
             "Sending SendBlock message to peer at {:?}",
             self.remote_addr
         );
-        send_block_msg(self, &msg.block)
+        send_block_msg(self, msg.block)
+    }
+}
+
+/// Handler for SendBlock message (sent by other actors)
```

Comment:
```suggestion
/// Handler for RequestBlock message (sent by other actors)
```

And add this message to the session documentation